go/oasis-test-runner/scenario/e2e/stake_cli.go

```diff
@@ -826,3 +828,9 @@ func (sc *stakeCLIImpl) genAmendCommissionScheduleTx(childEnv *env.Env, nonce in
 	}
 	return nil
 }
+
+func init() {
```

Comment:
Why do this in global init, I would much rather have this per test case.

Reply:
Agreed. I've made it per test case now.